Tests/biosql.ini.appveyor

```diff
@@ -22,6 +22,6 @@ testdb=biosql_test
 [pg]
 # Covers DBDRIVER="psycopg2" etc
 dbhost=localhost
-dbuser=postgres
-dbpasswd=Password12!
+dbuser=biosql_user
+dbpasswd=biosql_pass
```

Comment:
These changes are to match the account created in ``.appveyor.yml`` rather than the default account provided by AppVeyor, so the comment earlier is now out of date:

```python
# Values match https://www.appveyor.com/docs/services-databases/
```

I suggest moving that comment to the ``[mysql]`` section, and adding a new comment to the ``[pg]`` section.

Reply:
OK